moveit_ros/visualization/motion_planning_rviz_plugin/include/moveit/motion_planning_rviz_plugin/motion_planning_frame.hpp

```diff
@@ -331,8 +331,8 @@ void MotionPlanningFrame::waitForAction(const T& action, const ros::NodeHandle&
   ROS_DEBUG("Waiting for MoveGroup action server (%s)...", name.c_str());
 
   // in case ROS time is published, wait for the time data to arrive
-  ros::Time start_time = ros::Time::now();
-  while (start_time == ros::Time::now())
+  rclcpp::Clock start_time = rclcpp::Clock().now();
```

Comment:
Is it this working? The returning time from the clock().now() should be a rclcpp::Time
http://docs.ros2.org/bouncy/api/rclcpp/classrclcpp_1_1_clock.html#a6375208429c40bd9d6e84b2b32557182, am I wrong?

Reply:
I think you are right. Is it okay if I replace it with `auto`?

Reply:
Yes, we usually use *auto* for that

Reply:
0fc4f6629d8e7ffadaa13200233f1c483bb1f241